### PR TITLE
Add ssh key

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This PR:
* Fixes a bug due to other Ansible roles polluting the variable space and causing incorrect packages to be installed
* Adds code to install public and private ssh keys for the VNC user, if desired

## 💭 Motivation and Context

We need the correct packages to be installed, and we need to be able to transfer files to and from the Kali instance via sftp.

## 🧪 Testing

All the pre-commit hooks and the molecule tests pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -55,6 +57,42 @@
     group: "{{ username }}"
     mode: 0600
 
+#
+# Install an SSH keypair for the VNC user.  This will enable Guacamole
+# to perform file transfers to/from this instance.
+#
+# Note that Guacamole (v.1.0.0) currently only supports RSA keys.
```
Comment (dav3r) [inline tasks/main.yml]: `Guacamole v.1.0.0.` --> `Guacamole (guacd) v.1.1.0`

File: tasks/main.yml [hunk 2]
```diff
@@ -55,6 +57,42 @@
     group: "{{ username }}"
     mode: 0600
 
+#
+# Install an SSH keypair for the VNC user.  This will enable Guacamole
+# to perform file transfers to/from this instance.
+#
+# Note that Guacamole currently only supports RSA keys.
```
Comment (dav3r) [inline tasks/main.yml]: If we want to be more specific we could say "Note that Guacamole (guacd, as of v.1.1.0) only supports RSA keys."

**Review by dav3r — APPROVED** [review]
Now we're cooking! 🔥

File: tasks/main.yml (hunk 2 above)
Comment (jsf9k) [reply tasks/main.yml]: See commit d954293.

**Review by felddy — APPROVED** [review]
Good find DEV BOYO!

**Pull request merged**